app/views/people/_previous_roles.html.erb

```diff
@@ -1,5 +1,5 @@
 <% if person.has_previous_roles? %>
-  <section class="previous-roles govuk-!-padding-bottom-9">
+  <section class="previous-roles govuk-!-padding-bottom-9" dir="<%= page_text_direction %>" lang="<%= person.locale %>">
```

Comment:
This shouldn't be rtl, as it contains untranslated content in a left-to-right language.